UserService/UserService.Core/Domains/Entities/GetUserIsVerifiedRequest.cs

```diff
@@ -7,6 +7,6 @@ namespace UserService.Core.Domains.Entities
 {
     public class GetUserIsVerifiedRequest : IRequest<GetUserIsVerifiedResponse>
     {
-        public string ID { get; set; }
+        public int UserID { get; set; }
```

Comment:
not sure it is a good idea to change the name of the fields if other microservices/front  are consuming this api

Reply:
This method has never been called in Live, and not since 4th April in Dev, so I think we're OK.